mk/spksrc.spk.mk

```diff
@@ -14,8 +14,14 @@ TC = syno$(ARCH_SUFFIX)
 else
 SPK_ARCH = noarch
 SPK_NAME_ARCH = noarch
+ifeq ($(shell expr "$(TCVERSION)" \>= 7.0),1)
+SPK_TCVERS = dsm7
+OS_MIN_VER = 7.0-4000
+else
 SPK_TCVERS = all
 OS_MIN_VER = 3.1-1594
+endif
+ARCH_SUFFIX = -$(SPK_TCVERS)
```

Comment:
The ARCH_SUFFIX, I don't understand what it's for?

Reply:
This is for the noarch packages i.e. all (and new dsm7)
<package>-noarch-all-*.spk
<package>-noarch-dsm7-*.spk

Reply:
and ARCH_SUFFIX is used to define the WORK_DIR in spksrc.directories.mk
```
ifndef WORK_DIR
WORK_DIR = $(PWD)/work$(ARCH_SUFFIX)
endif
```

and it is mandatory to have different work dir for the two noarch builds.